pkg/internal/cluster/create/actions/kubeadminit/init.go

```diff
@@ -135,42 +114,3 @@ func (a *action) Execute(ctx *actions.ActionContext) error {
 	ctx.Status.End(true)
 	return nil
 }
-
-// matches kubeconfig server entry like:
-//    server: https://172.17.0.2:6443
-// which we rewrite to:
-//    server: https://$ADDRESS:$PORT
-var serverAddressRE = regexp.MustCompile(`^(\s+server:) https://.*:\d+$`)
```

Comment:
Since the 'rewrite' doesn't (seem to) happen, the `kind get kubeconfig` prints the container's host and port instead of 127.0.0.1 + the forwarded port. Or am I wrong?

Reply:
seems this is failing 
https://github.com/kubernetes-sigs/kind/pull/1029/files#diff-8e81c51b562c668c8c80420dd1917316R81
:/

can you open a bug  @maelvls ?